cloudferrylib/utils/utils.py

```diff
@@ -32,6 +32,10 @@
 import yaml
 from logging import config
 
+import cfglib
+
+# Initialize default config
+cfglib.initialize()
```

Comment:
What if `utils.py` never gets imported? I would suggest that you move all the logging-related logic into a separate module/class.